src/app/page.tsx

```
@@ -1,9 +1,9 @@
-'use client';
+"use client";
 
 import ImageCard from "@/components/ui/image-card";
 import { useQuery, useQueryClient } from "@tanstack/react-query";
 import { useEffect, useState } from "react";
```

ESLint

[error] 'useEffect' is defined but never used.

(@typescript-eslint/no-unused-vars)